Player/Console/PlayerCommand.php

```diff
@@ -113,10 +113,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
     }
 
-    private function createClient()
+    private function createClient($isDebug = false)
     {
         return new GuzzleClient([
             'cookies' => true,
+            'debug' => $isDebug,
```

Comment:
Done. Would you like that I explain this in the doc? I think this is useful information.